sklearn/gaussian_process/gpr.py

```diff
@@ -1,4 +1,7 @@
-"""Gaussian processes regression. """
```

Comment:
these should be added to the new classes that are added, not the files. For new files, the versionadded should be in each new class and function, so they show up in the docstring.